frontend/src/metabase/components/Schedule/components.tsx

```diff
@@ -165,25 +183,24 @@ export const SelectMinute = ({
 };
 
 export const AutoWidthSelect = (props: SelectProps) => {
-  const longestLabel = useMemo(
-    () => getLongestSelectLabel(props.data),
-    [props.data],
-  );
-  const maxWidth =
-    longestLabel.length > 15 ? "unset" : `${longestLabel.length + 0.85}rem`;
+  const maxWidth = useMemo(() => {
+    const longestLabel = getLongestSelectLabel(props.data);
+    const maxWidth = `${measureTextWidth(longestLabel) + 60}px`;
+    return maxWidth;
+  }, [props.data]);
```

Comment:
I'm assuming the idea here is that we want the select to be _just big enough_ to handle the largest possible item. However, this function is still just guessing at what the length might be and assumes a default font. I think the real answer here might be to use a custom dropdown component and use a ref to measure it's contents, but I'm going to punt this to @iethree and get his opinion

Reply:
I've made the `AutoWidthSelect` component a bit smarter so it takes the current font into consideration. Here's how the dropdowns look with Poppins as the font: 

![image](https://github.com/user-attachments/assets/34fabe36-745c-468d-aceb-f971b8e49c3c)

Previously, the dropdowns were narrower to a a slightly uncomfortable degree.